e2e/cypress/tests/support/win.d.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/// <reference types="cypress" />
+
+// ***************************************************************
+// Each command should be properly documented using JSDoc.
+// See https://jsdoc.app/index.html for reference.
+// Basic requirements for documentation are the following:
+// - Meaningful description
+// - Each parameter with `@params`
+// - Return value with `@returns`
+// - Example usage with `@example`
+// ***************************************************************
+
+declare namespace Cypress {
+    interface ApplicationWindow {
+
+        /**
+     * Reset all tracked selectors
+     * @example
+     *   win.resetTrackedSelectors();
+     */
```

Comment:
Please fix indentation. Unfortunately, this seemed to be not covered by our lint setup.

Reply:
Done, thanks